modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplatesTest.java

```diff
@@ -1759,26 +1771,29 @@ public void testBuildTemplatePortlet70() throws Exception {
 			"portlet", "foo.test", "--class-name", "Foo", "--liferayVersion",
 			"7.0");
 
-		_testContains(
-			gradleProjectDir, "bnd.bnd", "Export-Package: foo.test.constants");
 		_testContains(
 			gradleProjectDir, "build.gradle",
 			"apply plugin: \"com.liferay.plugin\"",
 			_DEPENDENCY_PORTAL_KERNEL + ", version: \"2.0.0");
 		_testContains(
 			gradleProjectDir,
 			"src/main/java/foo/test/constants/FooPortletKeys.java",
-			"public class FooPortletKeys");
+			"public class FooPortletKeys",
+			"public static final String FOO",
+			"\"foo.test_FooPortlet\";");
```

Comment:
hey Lawrence, this should be `foo_test_FooPortlet` to match the portal parameters.  So it seems the velocity is not working as expected here: `${package.replaceAll("\\.", "_")}_${className}Portlet`

Reply:
I just debuged it and it should be `"${package.replaceAll('\.', '_')}_${className}Portlet";`